### ISSUE TITLE
HTML and Gatherer flags

### ISSUE THREAD

**Issue by reimannsum** [billzorn/mtgencode#8]
when Using HTML and Gatherer flags mana_text is not properly handled giving `<img class="Mana-T">` instead of `<img class="mana-T">`

**Comment by billzorn** [comment]
This doesn't seem to be happening for me? I used flags:

```
./decode.py ../torch-rnn/s800r_t0.8.txt spoiler.html -v -d -html -g
```

**Comment by reimannsum** [comment]
weird I use `python2-7-11\python.exe mtgencode\decode.py output.txt spoiler.html -html -g` and it still happens to me. spoiler3 is your s800_t0.8 using my command, spoiuler 2 is using your command.
[spoiler2.html.txt](https://github.com/billzorn/mtgencode/files/318387/spoiler2.html.txt)

[output.txt](https://github.com/billzorn/mtgencode/files/318348/output.txt)
[spoiler.html.txt](https://github.com/billzorn/mtgencode/files/318351/spoiler.html.txt)
[spoiler3.html.txt](https://github.com/billzorn/mtgencode/files/318352/spoiler3.html.txt)

which is so weird, because after looking through the code there is no reason it would only effect the tap and untap symbols. I could understand if it effected effected every line where the name appeared in the card text as a overzealous capitalization function, but just the tap and untap?

**Comment by billzorn** [comment]
Oh I bet I know what's going on, I have the full NLTK smart capitalization thing installed, but there's a silent fallback if you don't. I'll turn it off and see if that lets me repro.

The tap symbol is a little weird because it seems like a manacost thing but it might be handled differently in the code.

**Comment by reimannsum** [comment]
yeah I think I noticed that, the tap and untap symbols weren't present in any of the mana symbol libraries
